services/venice-router/src/main/java/com/linkedin/venice/router/RouterServer.java

```diff
@@ -1,5 +1,8 @@
 package com.linkedin.venice.router;
 
+import static com.linkedin.venice.CommonConfigKeys.*;
```

Comment:
nit: not import all? Intellij keeps reverting the config

Reply:
freaking intellij